src/pages/errors.vue

```diff
@@ -36,49 +40,106 @@ const defaultDialogOptions = {
+        <template v-else>
+          Invalid Incorporation Application
+        </template>
       </span>
     </template>
+
     <template #content>
       <div class="text-[14px]">
-        <p>Your account is currently unable to access this Business. This may be because of the following: </p>
-        <div class="p-4">
-          <li>Your account is not authorized to access this Business — contact the Business owner to get access.</li>
-          <li>Your login session has timed out — please exit and then login again.</li>
-          <li>The specified Business Identifier is not valid.</li>
-        </div>
-        <p class="mb-4">
-          You can retry now, or you can exit and try to access this Business at another time.
-        </p>
-        <p class="mb-4">
-          {{ $t('text.dialog.error.contact') }}
-        </p>
-        <ContactInfo :contacts="registriesContact" />
+        <!-- Business Auth Error Dialog -->
+        <template v-if="showUnauthorizedAccess">
+          <!-- auth error for bootstrap business -->
+          <template v-if="isTempReg">
+            <p class="font-15">
+              Your account is currently unable to access this Incorporation Application.
+              This may be because of the following:
+            </p>
+            <ul>
+              <li>
+                Your account is not authorized to access this Incorporation Application &mdash; contact
+                the account owner to get access.
+              </li>
+              <li>Your login session has timed out &mdash; please exit and then login again.</li>
+              <li>The specified URL is not valid &mdash; exit and return to the Business Registry page.</li>
+            </ul>
+            <p class="mt-4">
+              You can retry now, or you can exit and try to access this Incorporation Application
+              at another time.
+            </p>
+          </template>
+          <!-- auth error for bootstrap business -->
+          <templale v-else>
+            <p>Your account is currently unable to access this Business. This may be because of the following:</p>
+            <ul>
+              <li>
+                Your account is not authorized to access this Business &mdash; contact the Business owner to get access.
+              </li>
+              <li>Your login session has timed out &mdash; please exit and then login again.</li>
+              <li>The specified Business Identifier is not valid.</li>
+            </ul>
+            <p class="mb-4">
+              You can retry now, or you can exit and try to access this Business at another time.
+            </p>
+          </templale>
+        </template>
+        <!-- Invalid Name Request Dialog -->
+        <template v-else>
+          <p class="font-15">
+            {{ nrErrorMessage }}
+          </p>
+          <p class="mt-4">
+            You can retry now, or you can exit and try to access this Incorporation Application
+            at another time.
```

Comment:
The Retry button should (but doesn't) reload all the data to see if it's now OK (like Filings UI does).

Reply:
This is still all that happens on Retry:

![image](https://github.com/user-attachments/assets/352b0351-224a-4c00-a2d7-7a09fb466dc1)

Reply:
Here are the network calls on Retry. It looks like this is enough to see if anything has changed (and therefore if it's worth retrying).

![image](https://github.com/user-attachments/assets/99d80adb-034d-4a2c-b050-159e5b856050)

Resolving.